itests/src/test/java/fi/metatavu/edelphi/test/ui/base/panel/QueryTestsBase.java

```diff
@@ -191,10 +300,28 @@ private void nextPage(Integer page) {
   private void finishQuery() {
     waitAndClick(String.format("input[name='%s']", "finish"));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "finish" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)